working-together/README.md

```diff
@@ -2,64 +2,6 @@
-- Agree Working Environment (Two keyboards? Text editor and other tools)
-- Agree Pairing Style (Time-based? Ping pong?)
-
-**During:**
-
-- Keep the chat going (Think aloud. Encourage/support)
-- Keep switching (Follow the pairing style)
-- Keep both involved ("Could we do this another way?")
-- Keep breaking ([Have a break, have a…](https://www.youtube.com/watch?v=fejBO1HZXVQ))
-- Keep checking in ("Could we search for a guide separately?")
-
-**After:**
-
-- Ask for Feedback (What did we do well? What could we have done better? It’ll feel weird, do it anyway)
-
-## Meeting Guidelines
-
-Here are some guidelines to help people feel included and respected during 
-meetings:
-
-* **Be kind:** Treat each other with respect.
-* **Remote matters:** Listen for and solicit input from remote members so 
-their voice is heard.
-  * Ensure the space is set up so everyone can see each 
-others' faces.
-  * Enunciate and speak loud enough so that remote members can hear you
-clearly.
-* **Introduce everyone:** Don't assume everyone has met before. Introductions 
-help break the ice and give an opportunity for voices to be heard.
-* **Pay Attention:** Be present with others during a meeting, but also engage 
-and communicate in the way you feel most comfortable.
-* **Be respectful of others' schedules:** Try to start and end meetings on time.
-* **Step up, step back:** After you speak, let others speak. If you are 
-speaking often, make sure others have had a chance to say something.
-* **Apply the Shine Theory:** Notice if someone is being cut off or ignored. Help 
-shine the light back on that contributor by repeating their idea, giving clear 
-support and credit to its source.
-* **Give credit:** Acknowledge input from others and highlight contributors. 
-Show encouragement and, if you agree with something, call it out. Ensure that 
-even the quiet voices are heard and supported.
-* **Why am I talking?:** Make sure that the thing you're about to say has not 
-already been said. If it has and has been ignored, clarify that you are 
-amplifying an under-recognized idea.
-* **One mic:** Let one person speak at a time, and let them complete their full 
-thought before responding. Don't interrupt.
-* **Clarify what you hear:** Use phrases like "What I heard you say was..." or 
-"My feedback for that is..."
-* **Speak as I:** Try to speak from the *I* perspective. Don't speak for others.
-* **Disagree:** You don't have to agree with everybody, but be aware of how you 
-frame your feedback. Criticize the idea, not the person.
-* **Everyone can participate:** Assuming someone doesn't want to participate 
-can be exclusionary. Give people the opportunity to decline an invitation.
-* **Speak up:** Follow up if team members treat others disrespectfully. If you 
-don't feel comfortable addressing someone directly, speak with your manager or Chief
-People Officer.
+* [Emotional Intelligence](emotional-intelligence.md)
```

Comment:
I am wondering if a title and related page that is more focused on inclusive culture fits better with the section on "Working Together"?

I love the writing and ideas. I am thinking it is broader than just working together though! And for working together, the important part of EQ is not just having it, but its outcomes: better decision making through a safer, happier and more inclusive work environment.

Reply:
Thanks Kane, I think that suggestion does hold this in a stronger light.

I had an intuition that the _Define Emotional Intelligence_ document was part of a greater "How to: Culture" guide, and none of the current top level guides seemed like the right fit. _Working Together_ was the closest match, though I dont think its the right fit either. 

Ill create a new top level document with a loosely held name of _Inclusive Culture_, and link to the _Define Emotional Intelligence_ from there. Other D&I task forces can build upon _Inclusive Culture_, beyond the EQ part.

Reply:
As of ebfd244f23e1489e6a8307c5dff8456964bd48d0 I had moved these documents to their own location in guides, and restored _Working Together_ to its prior state. :truck: